src/JobCloud.BE.Domain/Models/Technology.cs

```diff
@@ -0,0 +1,6 @@
+﻿using JobCloud.BE.Domain.Enums;
+
+namespace JobCloud.BE.Domain.Models
+{
+    public record Technology(Enums.Technology Name, Level TechnologyLevel);
```

Comment:
Question: How do records work in C#? What do they represent? Are they similar to `Record<X, Y>` in JS?

Reply:
Records are basically immutable classes and gives you ability to compare them not by references but values.